### ISSUE TITLE
Ensure KCard supports RTL

### ISSUE THREAD

**Issue by MisRob** [learningequality/kolibri-design-system#665]
## Blocked by

- https://github.com/learningequality/kolibri-design-system/pull/625

## Summary

`KCard` needs to support right-to-left languages. This was difficult to preview and test when developing solely in KDS, but as we're transitioning to Kolibri now, we can preview it more easily and ensure that everything works as expected.

One major area is the placement of the thumbnail area. If the thumbnail area appears on the right side in LTR, it should appear on the left side in RTL:

![Screenshot from 2024-06-18 21-12-00](https://github.com/learningequality/kolibri-design-system/assets/13509191/df59bc06-efb2-4a0c-8b65-c82a76b70bf2)

There may be more places to consider, and reviewing the card in all its layout modes in regard to RTL is one of this issue's tasks.

## Guidance
- [RTL language support](https://kolibri-dev.readthedocs.io/en/develop/i18n.html#rtl-language-support) Kolibri guide
- See [Bidirectionality](https://m2.material.io/design/usability/bidirectionality.html) guidelines to get a sense of what should be mirrored
- It is recommended to work on this in parallel to developing Kolibri cards based on `KCard` (such as https://github.com/learningequality/kolibri/issues/12317, https://github.com/learningequality/kolibri/issues/12318, or https://github.com/learningequality/kolibri/issues/12319)

## Out of scope
- `KCard` shouldn't be concerned about RTL for any elements that are provided by products via slots. It's the product's responsibility to ensure RTL for such elements.
  - Example: In the illustration above, `KCard` ensures that the `#aboveTitle` slot is rendered on the right side. But it is not (and cannot be) concerned whether icons provided to that slot from a product are mirrored.

## Acceptance criteria
- [ ] All `KCard` modes are reviewed for RTL
- [ ] It changes its layout appropriately when used in an RTL language
- [ ] It is tested in Kolibri in at least one place

**Comment by AllanOXDi** [comment]
Currently  this is what KCard  supports by default, no need to reinvent the wheel for now . We will need to do a thorough QA on Kolibri with the new cards that will be built especially with the card grid.  @MisRob and I agreed  to close this after testing  with folder cards and resource cards on Kolibri.
